python-sdk/tests_integration/sql/operators/test_dataframe.py

```diff
@@ -47,13 +47,17 @@
             "database": Database.MSSQL,
             "file": File(path=str(CWD) + "/../../data/homes2.csv"),
         },
+        {
+            "database": Database.MYSQL,
+            "file": File(path=str(CWD) + "/../../data/homes2.csv"),
+        },
         {
             "database": Database.DUCKDB,
             "file": File(path=str(CWD) + "/../../data/homes2.csv"),
         },
     ],
     indirect=True,
-    ids=["snowflake", "bigquery", "postgresql", "sqlite", "redshift", "mssql", "duckdb"],
+    ids=["snowflake", "bigquery", "postgresql", "sqlite", "redshift", "mssql", "duckdb", "mysql"],
```

Comment:
```suggestion
    ids=lambda d: d['database'],
```

Reply:
We can pass a function to `ids` and I think will be more useful than an entire constant list, since there some times we just want to comment out some databases, in which case we don't have to touch the `ids` list.

Reply:
done